test/test-mysql-binding/test.js

```diff
@@ -49,42 +54,46 @@ describe('On MySQL Database', () => {
     });
   });
 
-  it('fetches Unmapped credentials for mysql client', () => {
-    const binding = bindings.getBinding('MYSQL', 'mysql', {
+  it('fetches Unmapped credentials for odbc client in MySQL', () => {
```

Comment:
This seems to remove the tests for mysql instead of adding tests for odbc ?

Reply:
@mhdawson  Yes and below seems to remove odbc MySQL tests and adding tests for MySQL client. So the git history of this file is messed up. I can recreate it, but in that case I'll have force push :)

Reply:
Not quite sure I follow, can this PR just not be updated to avoid the removal?

Reply:
@mhdawson  You were right, just realized I accidentally replaced the tests for mysql client with the odbc ones.
I also did a bit of refactor on the test code to avoid duplications and being more readable by reusing the same assertion object accross MySQL clients.
https://github.com/nodeshift/kube-service-bindings/pull/40/commits/0fa1150cee19a61eb7730775969afc910567091c